cinema_service/settings.py

```diff
@@ -140,3 +141,28 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
+    "DEFAULT_THROTTLE_CLASSES": [
+        "rest_framework.throttling.AnonRateThrottle",
+        "rest_framework.throttling.UserRateThrottle",
+    ],
+    "DEFAULT_THROTTLE RATES": {"anon": "10/minute", "user": "30/minute"},
```

Comment:
There is a typo in the `REST_FRAMEWORK` settings: 'DEFAULT_THROTTLE RATES' should be 'DEFAULT_THROTTLE_RATES'. Ensure there is no space in the key name.